src/client/src/components/pages/participants/viewParticipants/index.jsx

```diff
@@ -27,6 +22,32 @@ export default class ViewParticpants extends Component {
   clear = () => {
     this.setState({ search: "" });
   };
+  
+// axios to make requests from backend.. 
+  componentDidMount = async () => {
+    try {
```

Comment:
I think it's better to make new function like <code>getData</code> and invoke it in componentDidMount function
because you will need it in the next steps of the project.

Reply:
Ok, I will do it in the next step